usr/share/rear/verify/VEEAM/default/400_verify_veeam.sh

```diff
@@ -0,0 +1,16 @@
+# 400_verify_veeam.sh
+# check that veeam vbrserver is actually available (ping)
+test "${VBRSRV}" || Error "Define VBRSRV (hostname or IP address) in $CONFIG_DIR/local.conf file"
+
+if test "$PING" ; then
+        if ping -c 1 "${VBRSRV}" >/dev/null 2>&1 ; then
+           Log "Veeam VBR Server: ${VBRSRV} seems to be up and running."
+        else
+           Error "Unable to contact Veeam VBR Server: ${VBRSRV}"
```

Comment:
The error message is not necessarily correct.
Actually the Veeam VBR Server could be accessible
but only not respond to a ping so a correct error message
should state what actually happened and not interpret that like
```
Error "Veeam VBR Server $VBRSRV does not respond to 'ping'"
```
so that the user knows what actually went wrong.
Perhaps the user had set PING.

Reply:
Hello,

The check command and error handling has been changed from ping only, to curl command, for reliable port connectivity check.

Regards